themes/thingweb/layouts/index.html

```diff
@@ -2,37 +2,99 @@
-	
-	<code>node-wot</code> also provides a browser bundle to visualize TDs and to enable the interaction with Things from the Web browser. A simple example can be found here: <a href="http://plugfest.thingweb.io/webui/" target="_blank">WebUI</a>.
-	<br />
-	It is <em>not</em> only a toolkit with some demo applications. <code>node-wot</code> enables you to create and experiment with real-world WoT applications. </p>
-						
-						</div>
+{{ partial "head.html" . }}
+
+<body>
+    {{ partial "header.html" . }}
+
+    <div class="holder">
+        <div class="wel-holder">
+            <div class="container">
+                <div class="wel-content text-center">
+                    <!-- <h1>Open-source Web of Things implementations</h1> -->
+                    <img src="img/others/what-line.png" class="img-fluid" />
+
+                    <div class="text-left">
+
+                        {{ .Content }}
+
+
+                        <p>
+                            Eclipse Thingweb™ offers components for making IoT solutions interoperable at scale by
+                            leveraging the W3C WoT
+                            standards, no matter if improving an existing solution or building a new one:
+                        </p>
+
+                        <ul>
+                            <li><b>Describe devices:</b> information, capabilities, and data schemas in a standardized
+                                format</li>
+                            <li><b>Integrate devices:</b> connectivity via various IoT protocols under a uniform
+                                interface</li>
+                            <li><b>Validate device descriptions:</b> consistent metadata for your devices and across
+                                directories</li>
+                            <li><b>Develop applications:</b>Web browser-like runtime for (headless) portable IoT apps
+                            </li>
+                        </ul>
+
+                        <p>
+                            Each component within the Eclipse Thingweb™ project is designed to be used independently
+                            from each other while working
+                            harmoniously together.
+                            This gives developers the flexibility to choose the specific tools they need for their IoT
+                            solution without sacrificing interoperability.
+                        </p>
+
+                        <div>
+                            <h2>node-wot</h2>
+
+                            <p>
+                                The Eclipse Thingweb™ <code>node-wot</code> is the official reference implementation of
+                                the Web of Things (WoT) Scripting API written in the <a
```

Comment:
for the official specification?

Reply:
Do you mean removing "official"? We can also just remove the whole thing starting with "written"?